lib/product-import.coffee

```diff
@@ -125,6 +122,16 @@ class ProductImport
           uniqueEnumUpdateActions = @_filterUniqueUpdateActions(enumUpdateActions)
           @_updateProductType(uniqueEnumUpdateActions)
       .then =>
+        # filter out products which do not have at least one SKU
+        originalLength = productsToProcess.length
+        productsToProcess = productsToProcess.filter(@_doesProductHaveSku)
+        filteredProductsLength = originalLength - productsToProcess.length
+
+        # if there are some products which do not have SKU
+        if filteredProductsLength
+          @logger.warn "Filtering out #{filteredProductsLength} product(s) which do not have SKU"
```

Comment:
I would not log it as caller can log it whenever is required through with help of summary data -> missingSKU count.